app/livechat/server/api/lib/officeHour.js

```diff
@@ -0,0 +1,12 @@
+import { hasPermissionAsync } from '../../../../authorization/server/functions/hasPermission';
+import { LivechatOfficeHour } from '../../../../models/server/raw';
+
+export async function findLivechatOfficeHours({ userId }) {
+	if (!await hasPermissionAsync(userId, 'view-l-room')) {
```

Comment:
We have a specific permission for this feature, why are you not using it?
- `view-livechat-officeHours`

Reply:
I used the same permission that was in the publication.